templates/workloads/services/backend/cf.yml

```diff
@@ -56,11 +56,11 @@ Resources:
             StartPeriod: {{.HealthCheck.StartPeriod}}
             Timeout: {{.HealthCheck.Timeout}}
 {{- end}}
-{{- if .Storage}}
+{{- if .Storage -}}
 {{include "mount-points" . | indent 10}}
-{{- end}}
+{{- end -}}
 {{include "sidecars" . | indent 8}}
-{{- if .Storage}}
+{{- if .Storage -}}
```

Comment:
oh does adding "-}}" remove an extra new line for these if blocks?

Reply:
yeah, that's right. It trims whitespace until the next directive. In these sections of template where there are multiple lines of interpolations one after another sometimes an extra newline can sneak in. The trailing `-}}` fixes that. I don't want to do it everywhere but it's helpful sometimes to clean up the rendered CF.